setuptools/extern/__init__.py

```diff
@@ -48,7 +48,7 @@ def load_module(self, fullname):
                 # on later Python versions to cause relative imports
                 # in the vendor package to resolve the same modules
                 # as those going through this importer.
-                if sys.version_info.major >= 3:
+                if sys.version_info >= (3, ):
```

Comment:
Can be changed to  `if not six.PY2`

Reply:
I think not, because this code is for loading the vendored packages and `six` is one of the vendored packages it's loading. I could be wrong, but I think it's not possible to use the vendored packages in `extern/__init__.py`

Reply:
Sry, I didn't know that six can't be used in this module.